pkg/serviceaccounts/controllers/docker_registry_service_test.go

```diff
@@ -229,8 +254,6 @@ func TestUpdateOldStyleSecretWithKey(t *testing.T) {
 		t.Fatalf("failed to become ready")
 	}
 
-	fakeWatch.Modify(registryService)
```

Comment:
this was because the test previous had to kick something and doesn't need to anymore?

Reply:
yes, proving the initial secret check should work regardless of the registry change